build-coordinator/src/main/java/org/jboss/pnc/coordinator/maintenance/DefaultRemoteBuildsCleaner.java

```diff
@@ -109,7 +114,8 @@ private Result deleteBuildsFromIndy(String buildContentId, String authToken) {
             foloAdmin.clearTrackingRecord(buildContentId);
             result = new Result(buildContentId, Result.Status.SUCCESS);
         } catch (IndyClientException e) {
-            String description = MessageFormat.format("Failed to delete temporary hosted repository identified by buildContentId {}.", buildContentId);
+            String description = MessageFormat.format("Failed to delete temporary hosted repository identified by buildContentId {}.",
```

Comment:
This won'ŧ work either. You have to use a number between the curly braces and null value is ok then. So it should be
```
... {0}.", buildContentId);
```